docker/resource_docker_container_funcs.go

```diff
@@ -43,8 +43,18 @@ func resourceDockerContainerCreate(d *schema.ResourceData, meta interface{}) err
 		Image:      image,
 		Hostname:   d.Get("hostname").(string),
 		Domainname: d.Get("domainname").(string),
+		Tty:        d.Get("tty").(bool),
+		OpenStdin:  d.Get("stdin_open").(bool),
 	}
 
+	// if value, ok := d["tty"]; ok {
+	// 	config.Tty = value.(bool)
+	// }
+
+	// if value, ok := d["stdin_open"]; ok {
+	// 	config.OpenStdin = d.Get("stdin_open").(bool)
+	// }
+
```

Comment:
```suggestion
```

Reply:
Thank you. it was my first version, but I had some issue.
Tested again and works, updated as suggested